src/test/java/api/items/ItemApiExamples.java

```diff
@@ -114,6 +117,12 @@ public void canCreateAnItemWithoutIDAndHRID()
 
     assertThat(createdItem.containsKey("id"), is(true));
 
+    assertThat(createdItem.containsKey("administrativeNotes"), is(true));
+
+    JsonArray createdNotes = createdItem.getJsonArray("administrativeNotes");
+
+    assertThat(createdNotes.getString(0), is(testNote));
+
```

Comment:
Personally, I might consider fetching this element as a list (empty when not present) and use something like Hamcrest's `contains` matcher as that checks that the only the expected elements are present.

Reply:
I had no idea such a thing existed.  I've modified the test, and will change the one in MODINV-580.

Reply:
Thank you